packages/rfw/lib/src/dart/text.dart

```diff
@@ -2215,6 +2223,18 @@ class _Parser {
         _advance();
         return false;
       }
```

Comment:
Should these identifiers be added to the reserved words list?

Reply:
Hmm. If we add them to the reserved words list then this becomes a breaking change.

Reply:
Wondering if something like https://github.com/flutter/flutter/issues/141833#issuecomment-1901153459 would be useful here as well (instead of changing the language). The implementation of number could be as follows.

```
final result = _fetch(...);
if (result is int) return result as int;
if (result is double) return result as double;
if (result == 'infinity') return double.infinity;
if (result == 'nan') return double.nan;
```

Reply:
That would similarly be a breaking change though.